portal-api/src/v1/gef/controllers/validation/facilities.js

```diff
@@ -21,6 +21,16 @@ const hasRequiredItems = (doc) => {
   if (doc.hasBeenIssued === true && !doc.coverEndDate) {
     required.push('coverEndDate');
   }
+  // TODO DTFS2-7171: Add logic to only push the below to required if the deal version exists and is > 1
+  if (!doc.facilityEndDateExists) {
+    required.push('facilityEndDateExists');
+  }
+  if (doc.facilityEndDateExists === true && !doc.facilityEndDate) {
+    required.push('facilityEndDate');
+  }
+  if (doc.facilityEndDateExists === false && !doc.bankReviewDate) {
```

Comment:
would `doc.facilityEndDateExists` be false here?

Reply:
Ah @AlexBramhill I see the confusion, and now I need to double check - I'm using Mark's variable naming, and I'm taking 'facilityEndDateExists = true' to mean it has an end date, and 'facilityEndDateExists = false' to mean they have selected no to the question 'does the facility have an end date' and so it instead has a bank review date. 

@MarRobSoftwire is this what you intended? 

I wonder if it would be clearer named 'hasFacilityEndDate' instead